config/dev.exs

```diff
@@ -0,0 +1,3 @@
+import Config
+
+config :onigumo, :http_client, HTTPoison
```

Comment:
Why don’t we use parentheses on function calls?

```suggestion
config(:onigumo, :http_client, HTTPoison)
```

Reply:
nepruď